inspirehep/records/marshmallow/authors/base.py

```diff
@@ -11,34 +11,66 @@
 from invenio_records_rest.schemas.json import RecordSchemaJSONV1
 from marshmallow import Schema, fields, post_dump
 
+from ..fields import NonHiddenRaw
 from .common import PositionSchemaV1
 
 
-class AuthorsMetadataSchemaV1(Schema):
-    """Schema for Authors records."""
-
-    _collections = fields.Raw(dump_only=True)
+class AuthorsMetadataRawPublicSchemaV1(Schema):
     acquisition_source = fields.Raw(dump_only=True)
     advisors = fields.Raw(dump_only=True)
     arxiv_categories = fields.Raw(dump_only=True)
     awards = fields.Raw(dump_only=True)
     birth_date = fields.Raw(dump_only=True)
     control_number = fields.Raw(dump_only=True)
     death_date = fields.Raw(dump_only=True)
-    deleted = fields.Raw(dump_only=True)
-    email_addresses = fields.Raw(dump_only=True)
+    # deleted = fields.Raw(dump_only=True)
+    # deleted_records = fields.Raw(dump_only=True)
+    email_addresses = NonHiddenRaw(dump_only=True)
     ids = fields.Raw(dump_only=True)
     inspire_categories = fields.Raw(dump_only=True)
+    # legacy_categories = fields.Raw(dump_only=True)
+    # legacy_creation_date = fields.Raw(dump_only=True)
+    # legacy_version = fields.Raw(dump_only=True)
     name = fields.Raw(dump_only=True)
     new_record = fields.Raw(dump_only=True)
-    positions = fields.Nested(PositionSchemaV1, dump_only=True, many=True)
-    should_display_positions = fields.Method(
-        "get_should_display_positions", dump_only=True
-    )
+    positions = fields.Raw(dump_only=True)
     project_membership = fields.Raw(dump_only=True)
+    public_notes = fields.Raw(dump_only=True)
     status = fields.Raw(dump_only=True)
     stub = fields.Raw(dump_only=True)
     urls = fields.Raw(dump_only=True)
+
+
+class AuthorsMetadataRawAdminSchemaV1(AuthorsMetadataRawPublicSchemaV1):
+    class Meta:
+        include = {"$schema": fields.Raw()}
+
+    _private_notes = fields.Raw(dump_only=True)
+    _collections = fields.Raw(dump_only=True)
+    deleted = fields.Raw(dump_only=True)
+    deleted_records = fields.Raw(dump_only=True)
+    legacy_categories = fields.Raw(dump_only=True)
+    legacy_creation_date = fields.Raw(dump_only=True)
+    legacy_version = fields.Raw(dump_only=True)
```

Comment:
why are all these fields (from `deleted` onwards) only available for admins? they are not private.

Reply:
they aren't used for now, and didn't wanna expose them since they are legacy related stuff, might go away in the future. Should I include them on the public one?

Reply:
I would say so, they are not private so no need to hide them.